lib/lookup.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
-const normgit = require('normalize-git-url');
-const util = require('util');
 const path = require('path');
+const util = require('util');
+
+const normgit = require('normalize-git-url');
```

Comment:
I think it's fine to have multiple commits while the PR is being reviewed, I assume @targos would squash once the PR was approved and ready to land (at least that's how things are often done in core).